### ISSUE TITLE
namespaced debug-bar panels

### ISSUE THREAD

**Issue by geminorum** [johnbillion/query-monitor#327]
debug-bar panel classes with namespace prefix wont display correctly due to improper escaping.

```
namespace geminorum\Example\DebagBar;
class Meta extends \Debug_Bar_Panel

...

<div class="qm qm-debug-bar" id="qm-debug_bar_geminorum\example\debagbar\meta">...</div>

...

<a href="#qm-debug_bar_geminorum\example\debagbar\meta">...</a>
```

p.s. on the old debug-bar plugin, simply replaceing `esc_attr()` with `sanitize_html_class()` works.

**Comment by johnbillion** [comment]
Cheers! I'll get that fixed.

**Comment by johnbillion** [comment]
Fixed in 4cf1e98
